handlers/getSignedUrl.js

```diff
@@ -0,0 +1,16 @@
+const { Storage } = require('@google-cloud/storage');
+const { getJsonFrom } = require('../common/api_adapter');
+const { bondBaseUrl, promiseHandler, Response, samBaseUrl } = require('../common/helpers');
+
+const getSignedUrl = promiseHandler(async req => {
+    const { bucket, object, provider } = req.body || {};
```

Comment:
Note: Currently this expects the (optional) provider to be passed directly. Is that what we want, or should we accept the DOS URI and compute the bond provider from that? It depends how formal that mapping is (or is intended to be).

Reply:
Talked to @gpolumbo-broad, tweaked as discussed.